packages/headless/src/features/search-parameters/search-parameter-serializer.ts

```diff
@@ -49,6 +78,20 @@ function serializeFacets(key: string, facets: Record<string, string[]>) {
     .join(delimiter);
 }
 
+function serializeRangeFacets(
+  key: string,
+  facets: Record<string, NumericRangeRequest[]>
+) {
+  return Object.entries(facets)
+    .map(([facetId, ranges]) => {
+      const value = ranges
+        .map(({start, end}) => `${start}${rangeDelimiter}${end}`)
+        .join(',');
```

Comment:
Can `start` and `end` contain `,` (e.g. 1,234,567)? or is this not allowed by typescript?

Reply:
Is it not possible to use commas between numbers. One could use `_`, but this is just syntactic sugar for the interpreter. Once the value is in memory, accessing the variable will return the number without the `_`. e.g.

```
const oneThousand = 1_000;
console.log(oneThousand); // 1000
```

Reply:
Hmm, I haven't seen this behavior, large numbers are returned without any commas,